pkgs/applications/misc/qtpass/default.nix

```
@@ -60,6 +62,14 @@ stdenv.mkDerivation rec {
     }"
   ];
 
+  installPhase = lib.optionalString stdenv.hostPlatform.isDarwin ''
+    runHook preInstall
+    mkdir -p $out/Applications
+    cp -r main/QtPass.app $out/Applications
+    makeWrapper $out/Applications/QtPass.app/Contents/MacOS/QtPass $out/bin/qtpass
```

```suggestion
    makeWrapper $out/Applications/QtPass.app/Contents/MacOS/QtPass $out/bin/qtpass ''${qtWrapperArgs[@]}
```

and `dontWrapQtApps = stdenv.hostPlatform.isDarwin;`

---

I should investigate how Qt loads plugins on darwin...